uiowa/uiowa
# Feature: Event Series content type

Resolves #4018

- [x] Unlink dates on event series
- [x] Admissions default nofollow robots settings for event, event series
- [x] Change summary field label and add clock icon to render

# How to test
`blt ds --site=admissions.uiowa.edu`
`yarn workspace uids_base build`
`drush @admissions.local uli`
- Add an event series and test out the fields (https://admissions.local.drupal.uiowa.edu/node/add/event_series)
  - Verify that nofollow is the default metatag setting for each page
  - Verify that webmaster, publisher, and editor can create a new page
- Add multiple events and (https://admissions.local.drupal.uiowa.edu/node/add/event) reference the event series page using the event series link field. 
  - Verify that nofollow is the default metatag setting for each page

**Story test**
As an editor, I can create an Event series page that pulls in published upcoming event content dates referencing it and is sorted chronologically.

<details>
<img width="1830" alt="Screen Shot 2021-08-25 at 3 44 42 PM" src="https://user-images.githubusercontent.com/1036433/130862049-27dea54d-151d-452c-b923-bb821ebda23a.png">
</details>
